projects/collection-editor-library/src/lib/services/editor/editor.service.ts

```diff
@@ -324,6 +324,34 @@ export class EditorService {
     };
     this.telemetryService.error(telemetryErrorData);
   }
-
-
+// this method is used to get all the contents in course/question inside every module and sub module
+  getContentChildrens() {
+    const treeObj = this.treeService.getTreeObject();
+    const contents = [];
+    treeObj.visit((node) => {
+      if (node.folder === false) {
+        contents.push(node.data.id);
+      }
+    });
+    return contents;
+  }
+  contentsCountAddedInLibraryPage() {
+    this.contentsCount = this.contentsCount + 1;
+  }
+checkContentsAdditionLimit() {
```

Comment:
can you rename the function to checkIfContensCanbeAdded() . Right now we are restricting only by having limit but later many things can come up. That is why generalizing the name